packages/teams-js/src/public/people.ts

```diff
@@ -50,21 +57,29 @@ export namespace people {
 
     let callback: selectPeopleCallbackFunctionType | undefined = undefined;
     let peoplePickerInputs: PeoplePickerInputs | undefined = undefined;
+    // eslint-disable-next-line strict-null-checks/all
```

Comment:
same here for null check

Reply:
I assume aaron's comment on the other one applies here as well, should be safe to remove